system/ee/ExpressionEngine/Addons/member/mod.member_settings.php

```diff
@@ -1671,13 +1671,18 @@ public function do_member_mini_search()
         $search_query = array();
 
         foreach ($_POST as $key => $val) {
+            // Don't search for site_id because it isn't on the members table
+            if ($key == 'site_id') {
+                continue;
+            }
+
             if ($key == 'group_id') {
                 if ($val != 'any') {
                     $search_query[] = " role_id ='" . ee()->db->escape_str($_POST['group_id']) . "'";
                 }
             } else {
                 if ($val != '') {
-                    $search_query[] = $key . " LIKE '%" . ee()->db->escape_like_str($val) . "%'";
+                    $search_query[] = ee()->db->escape_like_str($key) . " LIKE '%" . ee()->db->escape_like_str($val) . "%'";
```

Comment:
should this use `escape_str` instead?

Reply:
Yeah, good catch - I didn't realize there were two functions and let autocomplete fill in that function when I was typing it. I'm updating it and will resolve this conversation once it is pushed up.